### ISSUE TITLE
The process '/usr/bin/gpg' failed with exit code 2

### ISSUE THREAD

**Issue by dastrobu** [fwal/setup-swift#344]
**Describe the bug**

`setup-swift` fails on `ubuntu-latest` with `The process '/usr/bin/gpg' failed with exit code 2`. This looks like #77 reappeared. Full logs can be found below.

**Workflow configuration (please complete the following information):**
 - Platform (`runs-on`): `ubuntu-latest`
 - Swift version (`swift-version`): any, see 5.0 as example in logs below.

```
Run fwal/setup-swift@v1.11.0
  with:
    swift-version: 5.0
/usr/bin/gpg --import /home/runner/work/_temp/6f3cd87a-c90d-48ae-a6a4-9509bc4160d0
gpg: directory '/home/runner/.gnupg' created
gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
gpg: key D441C977412B37AD: public key "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" imported
gpg: key 9F597F4D21A56D5F: public key "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key 63BC1CFE91D306C6: public key "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key EF5430F071E1B235: public key "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key 7638F1FB2B2B08C4: public key "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" imported
gpg: key 925CC1CCED3D1561: public key "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key FAF6989E1BC16FEA: public key "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" imported
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
gpg: key F167DF1ACF9CE069: public key "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" imported
gpg: Total number processed: 9
gpg:               imported: 8
gpg:         new signatures: 1
/usr/bin/gpg --keyserver hkp://pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://pool.sks-keyservers.net
gpg: keyserver refresh failed: No name
Warning: An error occurred when trying to refresh keys from hkp://pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
/usr/bin/gpg --keyserver ha.pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://ha.pool.sks-keyservers.net
gpg: keyserver refresh failed: No name
Warning: An error occurred when trying to refresh keys from ha.pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
/usr/bin/gpg --keyserver keyserver.ubuntu.com --refresh-keys Swift
gpg: refreshing 8 keys from hkp://keyserver.ubuntu.com
gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" not changed
gpg: key FAF6989E1BC16FEA: "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" not changed
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 7638F1FB2B2B08C4: "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" not changed
gpg: key EF5430F071E1B235: "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 63BC1CFE91D306C6: "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 9F597F4D21A56D5F: "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key D441C977412B37AD: "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" not changed
gpg: Total number processed: 8
gpg:              unchanged: 8
/usr/bin/gpg --verify /home/runner/work/_temp/e10b5e90-937d-4a19-a17b-03c60e7bbdd5 /home/runner/work/_temp/7a6475e2-2771-4bf1-a368-ed6196ce9cd6
gpg: no valid OpenPGP data found.
gpg: the signature could not be verified.
Please remember that the signature file (.sig or .asc)
should be the first file given on the command line.
Error: Unexpected error, unable to continue. Please report at https://github.com/fwal/setup-swift/issues
The process '/usr/bin/gpg' failed with exit code 2
Stacktrace:
Error: The process '/usr/bin/gpg' failed with exit code 2
    at ExecState._setResult (/home/runner/work/_actions/fwal/setup-swift/v1.11.0/dist/index.js:995:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/fwal/setup-swift/v1.11.0/dist/index.js:978:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/fwal/setup-swift/v1.11.0/dist/index.js:872:27)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:430:11)
    at Socket.emit (events.js:210:5)
    at Pipe.<anonymous> (net.js:659:12)
```

**Comment by Samasaur1** [comment]
I'm also encountering this issue, and I can confirm that it is the same issue as #77. And if you look at the log excerpt that I've included below, you can see that despite setup-swift consulting a pool of key servers (as was discussed in the resolution to the original issue), it fails on most of them:
```
/usr/bin/gpg --keyserver hkp://pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://pool.sks-keyservers.net
gpg: keyserver refresh failed: Server indicated a failure
Warning: An error occurred when trying to refresh keys from hkp://pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
```
```
/usr/bin/gpg --keyserver ha.pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://ha.pool.sks-keyservers.net
gpg: keyserver refresh failed: Server indicated a failure
Warning: An error occurred when trying to refresh keys from ha.pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
```
and on the one server that works, none of the keys are updated:
```
/usr/bin/gpg --keyserver keyserver.ubuntu.com --refresh-keys Swift
gpg: refreshing 8 keys from hkp://keyserver.ubuntu.com
gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" not changed
gpg: key FAF6989E1BC16FEA: "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" not changed
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 7638F1FB2B2B08C4: "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" not changed
gpg: key EF5430F071E1B2[35](https://github.com/Samasaur1/SwiftIP/runs/5088383035?check_suite_focus=true#step:3:35): "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 63BC1CFE91D306C6: "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 9F597F4D21A56D5F: "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key D441C977412B37AD: "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" not changed
gpg: Total number processed: 8
gpg:              unchanged: 8
```

<details>
  <summary>Complete log</summary>
  
```
2022-02-07T04:53:56.7541964Z ##[group]Run fwal/setup-swift@v1
2022-02-07T04:53:56.7542189Z with:
2022-02-07T04:53:56.7542384Z   swift-version: 5.1
2022-02-07T04:53:56.7542584Z ##[endgroup]
2022-02-07T04:53:57.2030543Z [command]/usr/bin/gpg --import /home/runner/work/_temp/41047049-be27-47b4-88d1-aeddc7ec71d7
2022-02-07T04:53:57.2098387Z gpg: directory '/home/runner/.gnupg' created
2022-02-07T04:53:57.2099448Z gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
2022-02-07T04:53:57.2118386Z gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
2022-02-07T04:53:57.2119663Z gpg: key D441C977412B37AD: public key "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2173851Z gpg: key 9F597F4D21A56D5F: public key "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2185229Z gpg: key 63BC1CFE91D306C6: public key "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2197650Z gpg: key EF5430F071E1B235: public key "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2204686Z gpg: key 7638F1FB2B2B08C4: public key "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2211797Z gpg: key 925CC1CCED3D1561: public key "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2218666Z gpg: key FAF6989E1BC16FEA: public key "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:57.2317716Z gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
2022-02-07T04:53:57.2328671Z gpg: key F167DF1ACF9CE069: public key "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" imported
2022-02-07T04:53:57.2333695Z gpg: Total number processed: 9
2022-02-07T04:53:57.2334179Z gpg:               imported: 8
2022-02-07T04:53:57.2437461Z gpg:         new signatures: 1
2022-02-07T04:53:57.2438576Z [command]/usr/bin/gpg --keyserver hkp://pool.sks-keyservers.net --refresh-keys Swift
2022-02-07T04:53:57.4917181Z gpg: refreshing 8 keys from hkp://pool.sks-keyservers.net
2022-02-07T04:55:01.8269699Z gpg: keyserver refresh failed: Server indicated a failure
2022-02-07T04:55:01.8308710Z ##[warning]An error occurred when trying to refresh keys from hkp://pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
2022-02-07T04:55:01.8319468Z [command]/usr/bin/gpg --keyserver ha.pool.sks-keyservers.net --refresh-keys Swift
2022-02-07T04:55:01.8349264Z gpg: refreshing 8 keys from hkp://ha.pool.sks-keyservers.net
2022-02-07T04:56:21.9221359Z gpg: keyserver refresh failed: Server indicated a failure
2022-02-07T04:56:21.9233500Z ##[warning]An error occurred when trying to refresh keys from ha.pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
2022-02-07T04:56:21.9242917Z [command]/usr/bin/gpg --keyserver keyserver.ubuntu.com --refresh-keys Swift
2022-02-07T04:56:21.9283886Z gpg: refreshing 8 keys from hkp://keyserver.ubuntu.com
2022-02-07T04:56:25.6621123Z gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" not changed
2022-02-07T04:56:25.6625624Z gpg: key FAF6989E1BC16FEA: "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6636696Z gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6643144Z gpg: key 7638F1FB2B2B08C4: "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6649068Z gpg: key EF5430F071E1B235: "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6655726Z gpg: key 63BC1CFE91D306C6: "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6661743Z gpg: key 9F597F4D21A56D5F: "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6675250Z gpg: key D441C977412B37AD: "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" not changed
2022-02-07T04:56:25.6675619Z gpg: Total number processed: 8
2022-02-07T04:56:25.6675838Z gpg:              unchanged: 8
2022-02-07T04:56:26.1590532Z [command]/usr/bin/gpg --verify /home/runner/work/_temp/0dd3ba39-d930-456a-a78f-df98dd32a0be /home/runner/work/_temp/94ed0956-c297-403d-882d-d8ae733374e0
2022-02-07T04:56:26.1630085Z gpg: no valid OpenPGP data found.
2022-02-07T04:56:26.1631098Z gpg: the signature could not be verified.
2022-02-07T04:56:26.1631661Z Please remember that the signature file (.sig or .asc)
2022-02-07T04:56:26.1632696Z should be the first file given on the command line.
2022-02-07T04:56:26.1652446Z ##[error]Unexpected error, unable to continue. Please report at https://github.com/fwal/setup-swift/issues
The process '/usr/bin/gpg' failed with exit code 2
Stacktrace:
Error: The process '/usr/bin/gpg' failed with exit code 2
    at ExecState._setResult (/home/runner/work/_actions/fwal/setup-swift/v1/dist/index.js:995:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/fwal/setup-swift/v1/dist/index.js:978:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/fwal/setup-swift/v1/dist/index.js:872:27)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```
(from https://github.com/Samasaur1/SwiftIP/runs/5088383035)
</details>

**Comment by Samasaur1** [comment]
Compare this failing build to this successful build, on Ubuntu 18.04:

```
/usr/bin/gpg --keyserver hkp://pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://pool.sks-keyservers.net
gpg: keyserver refresh failed: Server indicated a failure
Warning: An error occurred when trying to refresh keys from hkp://pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
```
```
/usr/bin/gpg --keyserver ha.pool.sks-keyservers.net --refresh-keys Swift
gpg: refreshing 8 keys from hkp://ha.pool.sks-keyservers.net
gpg: keyserver refresh failed: Server indicated a failure
Warning: An error occurred when trying to refresh keys from ha.pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
```
Note that after the same two failures, we see this *successful* import of keys from the final key server:
```
/usr/bin/gpg --keyserver keyserver.ubuntu.com --refresh-keys Swift
gpg: refreshing 8 keys from hkp://keyserver.ubuntu.com
gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" not changed
gpg: key FAF6989E1BC16FEA: "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" not changed
gpg: key 925CC1CCED3D1561: 1 signature not checked due to a missing key
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
gpg: key 7638F1FB2B2B08C4: 1 signature not checked due to a missing key
gpg: key 7638F1FB2B2B08C4: "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" 1 new signature
gpg: key EF5430F071E1B235: "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 63BC1CFE91D306C6: "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" not changed
gpg: key 9F597F4D21A56D5F: 15 signatures not checked due to missing keys
gpg: key 9F597F4D21A56D5F: "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" 15 new signatures
gpg: key D441C977412B37AD: 1 signature not checked due to a missing key
gpg: key D441C977412B37AD: "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" 1 new signature
gpg: Total number processed: 8
gpg:              unchanged: 4
gpg:         new signatures: 18
gpg: no ultimately trusted keys found
```
and we see that the release is signed by one of the new keys downloaded
```
/usr/bin/gpg --verify /home/runner/work/_temp/240adf22-8e29-41c7-be68-96b87aa6e335 /home/runner/work/_temp/ca2f278c-7e54-45e5-8eca-78ed3cdd2dc2
gpg: Signature made Fri Aug 30 21:24:19 2019 UTC
gpg:                using RSA key 925CC1CCED3D1561
gpg: Good signature from "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: A62A E125 BBBF BB96 A6E0  42EC 925C C1CC ED3D 1561
```

<details>
<summary>View full log</summary>

```
2022-02-07T04:53:57.9222421Z ##[group]Run fwal/setup-swift@v1
2022-02-07T04:53:57.9223028Z with:
2022-02-07T04:53:57.9223335Z   swift-version: 5.0
2022-02-07T04:53:57.9223693Z ##[endgroup]
2022-02-07T04:53:58.7038978Z [command]/usr/bin/gpg --import /home/runner/work/_temp/34e3e582-28d3-46b4-bfcf-c1572ec232ec
2022-02-07T04:53:58.7417477Z gpg: directory '/home/runner/.gnupg' created
2022-02-07T04:53:58.7421440Z gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
2022-02-07T04:53:58.7427774Z gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
2022-02-07T04:53:58.7429983Z gpg: key D441C977412B37AD: public key "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7445875Z gpg: key 9F597F4D21A56D5F: public key "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7456087Z gpg: key 63BC1CFE91D306C6: public key "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7465450Z gpg: key EF5430F071E1B235: public key "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7503214Z gpg: key 7638F1FB2B2B08C4: public key "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7524816Z gpg: key 925CC1CCED3D1561: public key "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7596716Z gpg: key FAF6989E1BC16FEA: public key "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" imported
2022-02-07T04:53:58.7731434Z gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
2022-02-07T04:53:58.7740202Z gpg: key F167DF1ACF9CE069: public key "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" imported
2022-02-07T04:53:58.7742679Z gpg: Total number processed: 9
2022-02-07T04:53:58.7743272Z gpg:               imported: 8
2022-02-07T04:53:58.7743519Z gpg:         new signatures: 1
2022-02-07T04:53:58.7785140Z [command]/usr/bin/gpg --keyserver hkp://pool.sks-keyservers.net --refresh-keys Swift
2022-02-07T04:53:58.8291121Z gpg: refreshing 8 keys from hkp://pool.sks-keyservers.net
2022-02-07T04:55:18.9187256Z gpg: keyserver refresh failed: Server indicated a failure
2022-02-07T04:55:18.9227920Z ##[warning]An error occurred when trying to refresh keys from hkp://pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
2022-02-07T04:55:18.9241488Z [command]/usr/bin/gpg --keyserver ha.pool.sks-keyservers.net --refresh-keys Swift
2022-02-07T04:55:18.9277595Z gpg: refreshing 8 keys from hkp://ha.pool.sks-keyservers.net
2022-02-07T04:55:51.8829669Z gpg: keyserver refresh failed: Server indicated a failure
2022-02-07T04:55:51.8831861Z ##[warning]An error occurred when trying to refresh keys from ha.pool.sks-keyservers.net: Error: The process '/usr/bin/gpg' failed with exit code 2
2022-02-07T04:55:51.8856210Z [command]/usr/bin/gpg --keyserver keyserver.ubuntu.com --refresh-keys Swift
2022-02-07T04:55:51.8914216Z gpg: refreshing 8 keys from hkp://keyserver.ubuntu.com
2022-02-07T04:55:54.0987036Z gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" not changed
2022-02-07T04:55:54.1011565Z gpg: key FAF6989E1BC16FEA: "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" not changed
2022-02-07T04:55:54.1012013Z gpg: key 925CC1CCED3D1561: 1 signature not checked due to a missing key
2022-02-07T04:55:54.1014951Z gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
2022-02-07T04:55:54.1024168Z gpg: key 7638F1FB2B2B08C4: 1 signature not checked due to a missing key
2022-02-07T04:55:54.1034826Z gpg: key 7638F1FB2B2B08C4: "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" 1 new signature
2022-02-07T04:55:54.1041831Z gpg: key EF5430F071E1B235: "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:55:54.1048980Z gpg: key 63BC1CFE91D306C6: "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" not changed
2022-02-07T04:55:54.1052946Z gpg: key 9F597F4D21A56D5F: 15 signatures not checked due to missing keys
2022-02-07T04:55:54.1060637Z gpg: key 9F597F4D21A56D5F: "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" 15 new signatures
2022-02-07T04:55:54.1065428Z gpg: key D441C977412B37AD: 1 signature not checked due to a missing key
2022-02-07T04:55:54.1079130Z gpg: key D441C977412B37AD: "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" 1 new signature
2022-02-07T04:55:54.1112726Z gpg: Total number processed: 8
2022-02-07T04:55:54.1113082Z gpg:              unchanged: 4
2022-02-07T04:55:54.1113297Z gpg:         new signatures: 18
2022-02-07T04:55:54.1113547Z gpg: no ultimately trusted keys found
2022-02-07T04:56:03.7477884Z [command]/usr/bin/gpg --verify /home/runner/work/_temp/240adf22-8e29-41c7-be68-96b87aa6e335 /home/runner/work/_temp/ca2f278c-7e54-45e5-8eca-78ed3cdd2dc2
2022-02-07T04:56:05.4588050Z gpg: Signature made Fri Aug 30 21:24:19 2019 UTC
2022-02-07T04:56:05.4588424Z gpg:                using RSA key 925CC1CCED3D1561
2022-02-07T04:56:05.4589204Z gpg: Good signature from "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" [unknown]
2022-02-07T04:56:05.4589649Z gpg: WARNING: This key is not certified with a trusted signature!
2022-02-07T04:56:05.4590037Z gpg:          There is no indication that the signature belongs to the owner.
2022-02-07T04:56:05.4590768Z Primary key fingerprint: A62A E125 BBBF BB96 A6E0  42EC 925C C1CC ED3D 1561
2022-02-07T04:56:05.4698604Z [command]/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/3a7e033a-8b65-4994-b916-02f6d354c728 -f /home/runner/work/_temp/ca2f278c-7e54-45e5-8eca-78ed3cdd2dc2
2022-02-07T04:56:16.3598776Z [command]/opt/hostedtoolcache/swift-Ubuntu/5.0.3/x64/usr/bin/swift --version
2022-02-07T04:56:16.7919259Z Swift version 5.0.3 (swift-5.0.3-RELEASE)
2022-02-07T04:56:16.7920851Z Target: x86_64-unknown-linux-gnu
```
</details>

**Comment by Samasaur1** [comment]
Turns out this fails on Swift 4.2.x, 5.0.x, and 5.1.x, on ubuntu-latest. I don't understand GPG enough to explain why these are the only ones that don't work, but if you can figure out how to view the GitHub Actions log here: https://github.com/Samasaur1/SwiftIP/actions/runs/1804797343 it may be userful

**Comment by fwal** [comment]
Thanks, and sorry for the delay!
It seems like they've changed key servers, I'll get it fixed!

**Comment by fwal** [comment]
Fix released in [1.12.0](https://github.com/fwal/setup-swift/releases/tag/v1.12.0) 👍🏻
